### PR TITLE
fixed build error

### PR DESCRIPTION
**Pull request by S-Amman-W**
- fixed error that prevent build from occurring

### CODE REVIEW

**Review by Zinka010 — APPROVED** [review]
Looks good

File: backend/src/main/java/com/example/backend/BackendApplication.java [hunk 1]
```diff
@@ -108,5 +108,27 @@ public String getPlayerByID(@PathVariable String ID) throws ClassNotFoundExcepti
         }
     }
 
+    @PostMapping("/is_in_user_teams")
```
Comment (ad-world) [inline backend/src/main/java/com/example/backend/BackendApplication.java:111]: I think you should change this endpoint url, sounds a little bit misleading as to what the endpoint does. Maybe something like "add_to_user_team" makes more sense.

File: backend/src/main/java/com/example/backend/BackendApplication.java [hunk 2]
```diff
@@ -108,5 +108,27 @@ public String getPlayerByID(@PathVariable String ID) throws ClassNotFoundExcepti
...
+        try {
+            Connection connection = DriverManager.getConnection(url, username, password);
+            String insertRows = "";
+            for (int i = 0; i < playerIDs.length; i++){
+                insertRows += "(" + playerIDs[i] + "," + teamID + ")";
+                if (i != playerIDs.length - 1){
+                    insertRows += ",";
+                }
+            }
+            String readMessageQuery = "INSERT INTO IsInUserTeam VALUES " + insertRows;
```
Comment (ad-world) [inline backend/src/main/java/com/example/backend/BackendApplication.java:122]:
```suggestion
            String insertQuery = "INSERT INTO IsInUserTeam VALUES " + insertRows;
```

**Review by ad-world — CHANGES_REQUESTED** [review]

File: backend/src/main/java/com/example/backend/BackendApplication.java [hunk 3]
```diff
@@ -108,5 +108,27 @@ public String getPlayerByID(@PathVariable String ID) throws ClassNotFoundExcepti
...
+            String insertRows = "";
+            for (int i = 0; i < playerIDs.length; i++){
+                insertRows += "(" + playerIDs[i] + "," + teamID + ")";
+                if (i != playerIDs.length - 1){
+                    insertRows += ",";
+                }
+            }
+            String insertQuery = "INSERT INTO IsInUserTeam VALUES " + insertRows;
+            Statement statement = connection.createStatement();
+            ResultSet resultSet = statement.executeQuery(readMessageQuery);
```
Comment (ad-world) [inline backend/src/main/java/com/example/backend/BackendApplication.java:124]: have you tested that this works? afaik `executeQuery` is for SELECT statements, and `executeUpdate` is for DML (insert update delete) statements. i dunno if `executeQuery` works for inserts.

**Review by ad-world — CHANGES_REQUESTED** [review]

File: backend/src/main/java/com/example/backend/BackendApplication.java (hunk 3 above)
Comment (S-Amman-W) [reply backend/src/main/java/com/example/backend/BackendApplication.java]: Thanks for letting me know, have changed it accordingly now.

**Review by ad-world — APPROVED** [review]

**Pull request merged**